File: src/main/java/domain/ArmeCharacteristique.java
package domain;

import com.google.gson.annotations.SerializedName;

public class ArmeCharacteristique {
    @SerializedName("PA")
    private String pa;
    @SerializedName("Portée")
    private String po;
    @SerializedName("CC")
    private String coupCritique;
}

File: src/main/java/domain/ArmeCharacteristique.kt
package domain

import com.fasterxml.jackson.annotation.JsonProperty

data class ArmeCharacteristique(
    @JsonProperty("PA")
    private var pa: String = "",
    @JsonProperty("Portée")
    private val po: String = "",
    @JsonProperty("CC")
    private val coupCritique: String = ""
)